core/src/main/java/org/elasticsearch/index/engine/InternalEngine.java

```diff
@@ -352,6 +352,16 @@ private boolean checkVersionConflict(
             final long currentVersion,
             final long expectedVersion,
             final boolean deleted) {
+        if (op.versionType() == VersionType.FORCE) {
+            if (engineConfig.getIndexSettings().getIndexVersionCreated().after(Version.V_5_0_0_alpha1)) {
```

Comment:
I don't think this is the right hammer to use here, but sure - if you prefer it like this , I'm good.